### ISSUE TITLE
Not able to enable IMS registration on Etisalat in UAE

### ISSUE THREAD

**Issue by nihalali90** [kyujin-cho/pixel-volte-patch#169]
I am having Pixel 7 (running TQ3A.230705.001) - July update done - Etisalat sim card in UAE

I did below steps after disabling my esim. The etisalat sim is my physical sim.

1. Followed the steps of installing shizuku
2. Then "Pixel IMS", Turned on Enable VoLTE and VoWifi 
3. Restarted a number of times, not registered

Went to other issue threads, added IMS and it shows in the list, restarted and did the Pixel IMS registration again, still nothing.

![WhatsApp Image 2023-07-11 at 11 55 28 AM](https://github.com/kyujin-cho/pixel-volte-patch/assets/139212354/74bd5469-8f99-4cb6-a9aa-575885c11d3e)
![WhatsApp Image 2023-07-11 at 11 55 30 AM](https://github.com/kyujin-cho/pixel-volte-patch/assets/139212354/61be02a3-2dab-4fb1-b848-422d9ce159d0)
![WhatsApp Image 2023-07-11 at 11 55 29 AM (1)](https://github.com/kyujin-cho/pixel-volte-patch/assets/139212354/1ec3d3ac-1fc5-44a4-a1d6-67bdc489eb20)
![WhatsApp Image 2023-07-11 at 11 55 29 AM](https://github.com/kyujin-cho/pixel-volte-patch/assets/139212354/e32830a6-2203-49c8-bd79-82605432a1ae)
![WhatsApp Image 2023-07-11 at 11 55 28 AM (1)](https://github.com/kyujin-cho/pixel-volte-patch/assets/139212354/2337f334-fb6d-46c8-91d2-4c49dedc223e)

If anyone knows what else to do, let me know. Really wanted VoLTE to work

**Comment by moshenoah188** [comment]
write the IMS APN in capital latter.

**Comment by nihalali90** [comment]
> write the IMS APN in capital latter.

No change. Still cant get IMS registered

**Comment by afsc19** [comment]
Did you ever fix this?

**Comment by nihalali90** [comment]
> Did you ever fix this?

Nope. Havent tried since, bro. Got busy with work. Tried a lot. Failed

**Comment by fillwithjoy1** [comment]
Check with your carrier.
